new-readme.md

```diff
@@ -81,133 +83,141 @@ npm run dev
-```
+The created application is accessible using the http://localhost:3000/ and `http://<your_ip>:3000` URLs (you can [change your URL settings](https://github.com/reimagined/resolve/blob/master/docs/API%20References.md)).
+
+The `create-resolve-app` can is capable to create the following apps:
 
 * [**hello-world**](https://github.com/reimagined/resolve/tree/master/examples/hello-world)
 
-It's a simple empty example that can be used like **reSolve application boilerplate**.
+    An empty app that can be used as a template for any **reSolve** application (created by default).
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-hello-world-example.png"></p>
 
 * [**top-list**](https://github.com/reimagined/resolve/tree/master/examples/top-list)
 
-This example demonstrates **reactive read-models** that leads to client updating without page reloads. You can learn more about [read-models in documentation](https://github.com/reimagined/resolve/blob/master/docs/Read%20Model.md)
+    This example demonstrates how to work with the [reactive read-models](https://github.com/reimagined/resolve/blob/master/docs/Read%20Model.md) used to update information on the web page without reloading it.
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-top-list-example.png"></p>
 
 * [**todo**](https://github.com/reimagined/resolve/tree/master/examples/todo)
 
-This example demonstrates **simple work with view-models**. You can learn more in [ToDo List App Tutorial](https://github.com/reimagined/resolve/blob/master/docs/Tutorials/ToDo%20List%20App%20Tutorial.md) and [documentation](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md).
+    This example demonstrates how to work with the [view-models](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md). The app's creation process is detailed in the [ToDo List App Tutorial](https://github.com/reimagined/resolve/blob/master/docs/Tutorials/ToDo%20List%20App%20Tutorial.md).
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-todo-example.png"></p>
 
 * [**todo-two-levels**](https://github.com/reimagined/resolve/tree/master/examples/todo-two-levels)
 
-This example demonstrates work with view-models in case of **all events subscribtion**. Sometimes, for example for fast application prototyping, it's necessary to have view-model on client with all server events instead of part. There's a special feature called **wildcard**, that allows to subscribe to all events, you can learn more about that in [documentation](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md).
+    This example demonstrates how to work with view-models in the **all events subscribtion** case. Learn more about this case in the [View Model](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md) article. <!-- No info there -->
```

Comment:
Maybe we should we at least add a section with the "To be supplied" placeholder?

Reply:
Yes, it'd be good

Reply:
@TatyanaRyzh, Can you please do this, because I think I don't understand what's wrong with the **all events subscribtion** case and what should be the title